src/tomography/rules.mk

```diff
@@ -65,6 +65,24 @@ tomography_MODULES = \
 	$(FC_MODDIR)/tomography_model_iso.$(FC_MODEXT) \
 	$(EMPTY_MACRO)
 
+postprocess_TARGETS = \
```

Comment:
No, this is wrong. The variables are named after the directory. This scheme enables automatic build/clean-only-this-directory if you are in some subdirectory. Adding `*_TARGETS` that are not named after the directory (or at least added to that variable) will break that.